Packs/MailListener/Integrations/MailListenerV2/MailListenerV2.py

```diff
@@ -381,6 +387,13 @@ def get_email_as_eml(client: IMAPClient, message_id: int) -> dict:
     return mail_file[0] if mail_file else {}
 
 
+def _convert_to_bytes(data) -> bytes:
+    demisto.debug("Converting data to bytes.")
+    bytes_data = bytes(data)
+    demisto.debug("Converted data successfully.")
```

Comment:
I don't think we need this line. if the convention will fail, we will surly know anyway.